lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -82,6 +85,15 @@ def model_create_prm_any_baseline_building(model, building_type, climate_zone, h
       end
     end
 
+    if /prm/i =~ template
```

Comment:
Does this have to be on it's own or could it be part of the code that you modified above? Either way, do you think you could add some comments to explain what we're doing?

Reply:
Modified.